views/js/e2e/_helpers/commands/cleanupCommands.js

```diff
@@ -37,6 +37,9 @@ Cypress.Commands.add('deleteItem', (itemName) => {
     // Delete test
     cy.get(cleanupSelectors.itemsPage.itemDeleteButton).click();
 
+    //windows workaround
+    cy.wait(1000);
```

Comment:
Would be nice to avoid these. Does it not work if you add a `.should('exist')` before the `.click()`? Cypress is supposed to automatically retry until the element is present or it times out.

Reply:
I just removed it and it seems that I don't have this problem now.